README.md

```diff
@@ -39,69 +43,131 @@ Pre-built docker for popular AI frameworks. Easy to include heterogeneous hardwa
 
-- Ubuntu 16.04
-- Assign each server a static IP address. Network is reachable between servers.
-- Server can access the external network, especially need to have access to a Docker registry service (e.g., Docker hub) to pull the Docker images for the services to be deployed.
-- All machines' SSH service is enabled, share the same username / password and have sudo privilege.
-- Need to enable NTP service.
-- Recommend no Docker installed or a Docker with api version >= 1.26.
-- See [hardware resource requirements](https://github.com/Microsoft/pai/wiki/Resource-Requirement).
+[Submit a test job](examples/README.md#quickstart)
 
-#### 2 Deploy OpenPAI
+### FAQ
 
-If you have a cluster which contains more than 2 machine and want to deploy pai on it. Please choose ```Distributed deploy``` following.
+If there is any question durding deployment, check [here](docs/faq.md#deploy-and-maintenance-related-faqs) firstly.
 
-If you only have one mahince, and want to deploy pai on it. Please choose ```Single Box deploy``` following. 
+If FAQ doesn't resolve it, refer to [here](#get-involved) to ask question or submit an issue.
 
+## Train model
 
-##### 2.1 [Distributed deploy](./docs/pai-management/doc/distributed-deploy.md)
-##### 2.2 [Single Box deploy](./docs/pai-management/doc/single-box.md)
+### Why to schedule jobs
 
+Like all machine learning platforms, OpenPAI is a production tool. To maximize utilization, users is recommended to submit training jobs into OpenPAI, and OpenPAI will allocate resource to run it. If there are too many jobs, some jobs may wait in the queue. This approach is different with on dedicated servers for each person, and it needs a bit more knowledge about how to submit/manage your jobs on OpenPAI.
 
+OpenPAI also supports to allocate dedicated resource for users, and user can use SSH or Jupyter on assigned environment like a standalone server. Refer [here](examples/jupyter/README.md) to use OpenPAI like this way. Though it's not efficient to use resource, but it also saves cost on setup and managing environments on physical servers.
 
-## How to use
-### How to train jobs
-- How to write OpenPAI jobs
-    - [Quick start: how to write and submit a CIFAR-10 job](./examples/README.md#quickstart)
-    - [Write job from scratch in depth](./docs/job_tutorial.md)
-    - [Learn more example jobs](./examples/#offtheshelf)
-- How to submit OpenPAI jobs
-    - [Submit a job in Web Portal](./docs/submit_from_webportal.md)
-    - [Submit a job in Visual Studio](https://github.com/Microsoft/vs-tools-for-ai/blob/master/docs/pai.md)
-    - [OpenPAI VS Code Extension](./contrib/pai_vscode)
-- How to run AutoML jobs on OpenPAI
-    - [Submit a job in Neural Network Intelligence](https://github.com/Microsoft/nni/blob/master/docs/PAIMode.md)
-- How to request on-demand resource for in place training
-    - [Launch a jupyter notebook and work in it](./examples/jupyter/README.md)
+### Train first model
```

Comment:
how about change to "Get familiar with Job by training example model"

Reply:
changed to "Train your first model on OpenPAI". IMO, user needs to train model, but "get familiar with job" is not direct link to their needs.